### PR TITLE
Basic layout: Navigation Bar

### PR DESCRIPTION
**Pull request by valeriafurin**
Hi there, 
I built the Navigationbar with a responsive expandable Burgermenu. The Links are not linked yet, will handle that later.

Best

### CODE REVIEW

File: src/GlobalStyle.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+export const SewLayout = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 100vh;
+  align-items: center;
+  justify-content: center;
+`;
+
+// const sewBox = styled.div`
```
Comment (barsukov) [inline src/GlobalStyle.tsx:11]: please delete the comments

File: src/components/Burger/Burger.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { StyledBurger } from "./Burger.styled";
+
+type BurgerProps = {
+  open: boolean;
+  setOpen: (open: boolean) => void;
+};
+
+const Burger: React.FunctionComponent<BurgerProps> = ({ open, setOpen }) => {
+  const clickHandler = () => setOpen(!open);
```
Comment (barsukov) [inline src/components/Burger/Burger.tsx:10]: nice solution 👍

File: src/pages/HomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Home = () => {
+  return (
+    <div
+      style={{
```
Comment (barsukov) [inline src/pages/HomePage.tsx:6]: it is better to use the styled component in this case.

File: src/pages/SkirtCalculatorPage.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const SkirtCalculator = () => {
+  return (
+    <div
+      style={{
```
Comment (barsukov) [inline src/pages/SkirtCalculatorPage.tsx:6]: please make this div as a styled component

File: src/pages/FabricCalculatorPage.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const FabricCalculator = () => {
+  return (
+    <div
+      style={{
```
Comment (barsukov) [inline src/pages/FabricCalculatorPage.tsx:6]: styled component

File: src/pages/AboutPage.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const About = () => {
+  return (
+    <div
+      style={{
```
Comment (barsukov) [inline src/pages/AboutPage.tsx:6]: styled component

**Review by barsukov — CHANGES_REQUESTED** [review]
few comments that you can fix before merge but the next task would be nice to integrate the CI pipeline and write your first jest tests.

File: src/GlobalStyle.tsx (hunk 1 above)
Comment (valeriafurin) [reply src/GlobalStyle.tsx:11]: Sure, forgot to do that

File: src/components/Burger/Burger.tsx (hunk 2 above)
Comment (valeriafurin) [reply src/components/Burger/Burger.tsx:10]: Thanks!

File: src/pages/HomePage.tsx (hunk 3 above)
Comment (valeriafurin) [reply src/pages/HomePage.tsx:6]: Alright, will change that. Was not sure about layout, so took this as a placeholder.

**Review by barsukov — APPROVED** [review]
🥇 good work

**Pull request merged**